recipes/pdfplumber/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pdfplumber" %}
+{% set version = "0.5.25" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pdfplumber-{{ version }}.tar.gz
+  sha256: ea52e877d6d2b9ac088a59f0feb1379ea0d989face5dd5c36e865b1763667dc2
+
+build:
+  number: 0
+  entry_points:
+    - pdfplumber = pdfplumber.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
What's the reason for skipping Windows? If just a missing dependency, we can ignore that failure and put this back to `noarch: python` which would be a good win.

Reply:
Correct it's cause of the missing dependency. I restored noarch python.